src/common_swagger_api/schema/groups.clj

```diff
@@ -60,7 +64,7 @@
      (describe Long (str "The date and time the " group-descriptor " was created (ms since epoch)"))
 
      :created_by
-     (describe String (str "The ID of the subject who created the " group-descriptor))
+     (describe subjects/Subject (str "The details of the subject who created the " group-descriptor))
```

Comment:
I was just testing this out with the teams UI hitting my local terrain and realized I have a schema violation with this if the user requests to see a single group's details (as opposed to fetching a list of groups).  The simplest thing to do would be to update this to something like
```
(s/conditional map? subjects/Subject :else String)
```
or I could update all cases to include the creator `Subject` in `created_by`.

Reply:
Another option would be to not update the `created_by` key and instead add a new key with the creator details.  I might go that route. 🤔